src/scenes/Species/SpeciesProjectsTable.tsx

```diff
@@ -86,17 +101,18 @@ export default function SpeciesProjectsTable({
     if (removedProjectsIds) {
       updatedResults =
         searchResults?.filter((sResults) => {
-          return !removedProjectsIds?.includes(Number(sResults.id));
+          return !removedProjectsIds?.includes(Number(sResults.participantProjectSpeciesId));
         }) || [];
     }
     if (addedProjectsIds) {
       const newProjects = addedProjectsIds?.map((id) => {
         return {
-          id: id,
-          submissionStatus: 'Not Submitted',
-          projectName: allProjects?.find((proj) => proj.id === id)?.name,
+          participantProjectSpeciesId: -1,
```

Comment:
I made this id to be the projectId because if then this entrance is removed, it will use that id to remove it, and if we have more than one added project with id -1, it will remove all of them

Reply:
Also when removing from table it uses the participantProjectSpeciesId to add to the removed list, but the added list uses projectId, so -1 will never exist in the addedProjects

Reply:
I'm not sure I follow, is the project ID used below sufficient? These entries are technically new "participant project species" entities, so they will have a `projectId` but no `participantProjectSpeciesId`

Reply:
The problem ocurs when you want to remove a new "participant project species" entity. That entity will be in the "added" list, with its `projectId`, but no `participantProjectSpeciesId`, because it doesn't exist in the BE. So if we want to remove it, we need the projectId of it, but the "removed list" uses the `participantProjectSpeciesId`, not the `projectId`.  (reference Line 62 of SpeciesEditView.tsx)